### DISCUSSION TITLE
5.12.3 (2022-07-27)

### DISCUSSION THREAD

**Discussion by manusa** [fabric8io/kubernetes-client#4306] [Announcements]
#### Bugs
- Fix #3969: relist will not trigger sync events
- Fix #4049: properly populate exception metadata with resource information if available
- Fix #4222: backport of #4082 - to not process events until the cache is complete
- Fix #4246: KubernetesClientException is swallowed in LeaderElector
- Fix #4295: Configure SnakeYaml to ignore converting timestamps to Date objects

<hr /><em>This discussion was created from the release <a href='https://github.com/fabric8io/kubernetes-client/releases/tag/v5.12.3'>5.12.3 (2022-07-27)</a>.</em>

**Comment by rdesgroppes** [comment]
I confirm redundant notifications are now gone, many thanks!

**Reply by manusa** [reply]
Good news, thanks for the feedback, we really appreciate it.
